src/oaklib/implementations/gilda.py

```diff
@@ -41,8 +42,20 @@ def annotate_text(
         import gilda
 
         for match in gilda.ground(text):
+            term_id_split = match.term.id.split(":")
```

Comment:
It's highly advisable to use the Bioregistry for doing these kinds of normalizations rather than re-implementing it again and again which could potentially introduce bugs.

Reply:
I think the fix is for gilda to always return CURIEs?

Reply:
@cthoyt - did you see the parent issue? #423